modules/ROOT/pages/dataweave-troubleshoot.adoc

```diff
@@ -0,0 +1,105 @@
+= Troubleshooting
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, anypoint, esb, transform, transformer, format, aggregate, rename, split, filter convert, xml, json, csv, pojo, java object, metadata, dataweave, data weave, datamapper, dwl, dfl, dw, output structure, input structure, map, mapping
+
```

Comment:
I recommend that any topic at this level that contains subtopics should include some text that introduces what those subtopics are, providing a context, and tell why the audience cares (what is the user goal or advantage).

Reply:
Is this the topic hierarchy intended?

=Troubleshooting
==Troubleshoot a Failing Script
==Exception: org.mule.weave.v2.exception.UnsupportedTypeCoercionException
=== Function Does Not Accept Null Argument
=== MIME Type Is Not Set

Try to make topic titles that are at the same hierarchical level parallel. 
I'm not sure you intend the "Exception" titled text to be a peer to the failing script text.
And are the following two topics (Function..., MIME...) subordinate to "Exception"?